homeworks/week4/Tam-Tran.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import random
+
+class Interval:
+    def __init__(self, low, high):
+        self.low = low
+        self.high = high
+
+    def len(self):
+        return self.high - self.low
+
+# time complexity: O(n)
+# space complexity: O(1)
+def generate_random_value(intervals):
```

Comment:
Similar to Vu's solution.

So my comment to his solution applies to yours https://github.com/P2PLearning/DSA/pull/28/files